scipy/stats/tests/test_stats.py

```diff
@@ -5792,6 +5799,93 @@ def test_weights_masked_1d_array(self):
         check_equal_gmean(a, desired, weights=weights, rtol=1e-5)
 
 
+class TestPowMean:
+    def test_1d_list(self):
+        #  Test a 1d list
+        a, p = [10, 20, 30, 40, 50, 60, 70, 80, 90, 100], 3.5
+        desired = 69.16258790433133
```

Comment:
Re: conversation above, external reference is preferable, but if there really are no published values, I'd suggest:

```suggestion
        a, p = np.array([10, 20, 30, 40, 50, 60, 70, 80, 90, 100]), 3.5
        desired = 1/len(a) * np.sum(a**p)^(1/p)
```
or similar, if I made a mistake.
When the definition quite simple like this, I think it's fine/good to repeat it (as concisely as possible) right in a test. If you need it for several tests, it would also be OK to define a simplified reference function:
```python3
    def pmean_reference(self, a, p):
        return 1/len(a) * np.sum(a**p)^(1/p)
```
as use it as needed.

Reply:
Agreed!

Reply:
Ok. Power mean can be written:
```
def pmean_reference(self, a, p):
    return (np.sum(a**p) / len(a))^(1/p)
```
and its weighted version:
```
def pmean_reference(self, a, p, weights):
    return (np.sum(weights * a**p) / np.sum(weights))^(1/p)
```

Reply:
Ok, I have found a solution to manage dimensions. @mdhaber @tupui

Reply:
Now, tests are OK. @mdhaber @tupui

Reply:
Thanks for the update.